### ISSUE TITLE
JPA

### ISSUE THREAD

**Issue by slAvIk159** [MaksKrysevych/carshop#3]
https://www.baeldung.com/the-persistence-layer-with-spring-data-jpa
Розбирався в такому?

**Comment by MaksKrysevych** [comment]
> https://www.baeldung.com/the-persistence-layer-with-spring-data-jpa Розбирався в такому?

так, але мало користувався цим, коли проходив курси в єпамі, було на вибір або JPA або Hibernate, то більше 2 використовував

**Comment by slAvIk159** [comment]
раджу спробувати щось пописати на JPA, або навіть мігранути цей проект на JPA. з практики частіше з JPA працював
